### DISCUSSION TITLE
ROOT/SUDO is required to run this script

### DISCUSSION THREAD

**Discussion by H4RUM4K1** [LSPosed/MagiskOnWSALocal#438] [Q&A]
So I cloned the repo to my PC, get the MagiskONWSALocal folder, and everything inside it, and went to the scripts folder. I opened BASH here and type
```shell
$ ./build.sh -- arch x64 --release-type WIF  --magisk-ver stable --gapps-brand none --gapps-variant pico --remove-amazon --root-sol magisk
``` 

Somehow I've got the following error and can't find any solution to it online, can anyone please help me

```shell
bash.exe: warning: could not find /tmp, please create!
INFO: Architecture: x64
INFO: Release Type: retail
INFO: Magisk Version: stable
INFO: GApps Brand: MindTheGapps
INFO: GApps Variant: pico
INFO: Root Solution: magisk
Build: RELEASE_TYPE=Retail
ROOT/SUDO is required to run this script
Build: an error has occurred, exit

Cleanup Work Directory
Cleanup Temp Directory


```

**Comment by Almighty188** [comment]
Before you do the command, "su" first.

**Reply by H4RUM4K1** [reply]
Very sorry for being dumb. At first, I thought I have to install it on Windows like a normal app, but after reading through the guide again I realized that this is not built, and I have to build it myself on Linux before installing it on Windows. Thanks for the help

**Reply by Almighty188** [reply]
Yeah, the guide does not really tell you which things are done on the Linux side and which are done on the Windows side unless you are familiar with both.
